Js/videoLoader.js

```diff
@@ -0,0 +1,15 @@
+/**
+ *adds video i for each div there are in the html file

```

Comment:
For photos and videos, if the number of photos/videos exceed the number of divs, how will the videos which do not fit be viewed?